core/src/main/scala/org/apache/spark/sql/delta/Checkpoints.scala

```diff
@@ -290,18 +290,39 @@ object Checkpoints extends DeltaLogging {
     // log store and decide whether to use rename.
     val useRename = deltaLog.store.isPartialWriteVisible(deltaLog.logPath, hadoopConf)
 
-    val checkpointRowCount = spark.sparkContext.longAccumulator("checkpointRowCount")
-    val numOfFiles = spark.sparkContext.longAccumulator("numOfFiles")
+    val sessionConf = spark.sessionState.conf
+    val maxActionsPerFile = sessionConf.getConf(DeltaSQLConf.DELTA_CHECKPOINT_MAX_ACTIONS_PER_FILE)
```

Comment:
validate config value `maxActionsPerFile`?

Reply:
Just add a validation to the conf itself? Just > 0 or any more validation you had in mind?

Reply:
If we can add it to the conf that is fine. Yeah just a postive number check.

Reply:
Add positive check